webapp/channels/src/root.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
 
-    <meta name='viewport' content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0'>
+    <meta name='viewport' content='width=device-width, initial-scale=1.0'>
```

Comment:
This fixes the issue of `Ensures <meta name="viewport"> does not disable text scaling and zooming.` See https://accessibilityinsights.io/info-examples/web/meta-viewport/

Reply:
thank you for removing this. user-scalable=0 is not a recommended approach